huggingface/pytorch/tgi/docker/1.4.5/Dockerfile

```diff
@@ -0,0 +1,265 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.75 AS chef
+WORKDIR /usr/src
+
+ARG CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
+
+FROM chef as planner
+COPY Cargo.toml Cargo.toml
```

Comment:
Could you add below line to use pinned versions?

```
COPY Cargo.lock Cargo.lock
```

Reply:
Yep updating now